### PR TITLE
[RFR] Clean dependencies

### PR DESCRIPTION
**Pull request by Kmaschta**
Removed packages:
- thunkify
- uid
- source-map-support
- style-loader
- co-fs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,37 +21,9 @@
     "koa-helmet": "1.0.0",
     "koa-mount": "1.3.0",
     "koa-route": "2.4.2",
-    "moment": "2.11.2",
```
Comment (djhi) [inline package.json]: moment might be useful in the API too. I suggest we keep it in deps.

Comment (brikou) [inline package.json]: IMO having unused deps in package.json is a not good thing. This package.json is already very fat.

Comment (Kmaschta) [inline package.json]: For now, this is useless 12Ko for the API.

**Comment by Kmaschta** [comment]
Removed packages listed in description

**Pull request merged**